### ISSUE TITLE
cannot find rule: redundant-build-tag

### ISSUE THREAD

**Issue by comerc** [mgechev/revive#1265]
**Describe the bug**

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file:

```shell
# flags

revive ...
```

```toml
# config file
```

**Expected behavior**

**Logs**

**Desktop (please complete the following information):**
 - OS: [e.g. Ubuntu 18.04]
 - Version of Go

**Additional context**

**Comment by alexandear** [comment]
@comerc Thank you for reporting the issue! However, could you provide more details to help us better understand the problem? Specifically:

1. The contents of your configuration file (or at least the relevant parts).
2. The exact output/error message you received.
3. The version of Go you are using (`go version`).

**Comment by alexandear** [comment]
Related to #1266

**Comment by alexandear** [comment]
@chavacava I think we can close the issue, because the rules `use-errors-new` and `redundant-build-tag` work as expected.

The contents of revive.go:
```go
//go:build ignore
// +build ignore

package revive

import "fmt"

func _() error {
	return fmt.Errorf("failure")
}
```

The contents of revive.toml:
```toml
[rule.use-errors-new]
[rule.redundant-build-tag]
```

Running the latest revive:
```sh
$ go run github.com/mgechev/revive@latest -config revive.toml revive.go
go: downloading github.com/mgechev/revive v1.8.0
revive.go:9:9: replace fmt.Errorf by errors.New
revive.go:2:1: The build tag "// +build" is redundant since Go 1.17 and can be removed
```
